<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorDialog</class>
 <widget class="QDialog" name="ColorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>509</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>509</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>509</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>361</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2"/>
  </widget>
  <widget class="QLabel" name="label_b">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>350</y>
     <width>33</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>33</width>
     <height>36</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>33</width>
     <height>36</height>
    </size>
   </property>
   <property name="text">
    <string>   B</string>
   </property>
  </widget>
  <widget class="QWidget" name="colorPreviewWg" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_b">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>350</y>
     <width>60</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>36</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_g">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>60</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>36</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_r">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>60</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>36</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_r">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>33</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>33</width>
     <height>36</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>33</width>
     <height>36</height>
    </size>
   </property>
   <property name="text">
    <string>   R</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_g">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>33</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>33</width>
     <height>36</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>33</width>
     <height>36</height>
    </size>
   </property>
   <property name="text">
    <string>   G</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>420</y>
     <width>270</width>
     <height>38</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="cancelBtn">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>36</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>16</width>
        <height>36</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okBtn">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>36</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="CloseButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>13</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CloseButton</class>
   <extends>QPushButton</extends>
   <header location="global">CloseButton/closebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
